src/AppBundle/Controller/MainController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use AppBundle\Entity\Todo;
+use Symfony\Component\HttpFoundation\Request;
+
+class MainController extends Controller
+{
+    /**
+     * @Route("/main", name="main"))
+     */
+    public function mainAction()
+    {   
+        if (!($this->container->get('security.authorization_checker')->isGranted('ROLE_USER'))) {
+            return $this->redirectToRoute('homepage');
+        }
+        $user = $this->container->get('security.token_storage')->getToken()->getUser();
```

Comment:
I wasn't aware, this is much quicker! Thanks! Definitely saves coding and my brain memory.